packages/uikit-workshop/webpack.config.js

```diff
@@ -373,7 +350,6 @@ module.exports = function(apiConfig) {
         new MiniCssExtractPlugin({
           filename: `[name].css`,
           chunkFilename: `[id].css`,
-          allChunks: true,
```

Comment:
@sghoweri could you please evaluate whether it's (still) necessary? it has been added with https://github.com/pattern-lab/patternlab-node/commit/318801a4ac10eb7263a09ab948c5cc272a922ce9#diff-8c9b7dba764c6bd34aaec36a6a42e7b69d9a46b19712a47706355bb8616d3319R154 but is now marked as not being a parameter (any more)

Reply:
If the build is still ok and the preview loads up correctly after building the ui-kit I would assume we do not need the parameter anymore.

Reply:
I've tested it, it's still okay.